docs(trace): traceability kit release for 23.09 (3.2.0)

mhellmeier opened the pull request:
This pull request updates the Traceability KIT to be compatible with the Catena-X release 3.2.0.

SebastianBezold commented on docs-kits/kits/Traceability Kit/Software Development View/page_software-development-view.md:
```diff
@@ -386,55 +442,284 @@ If no access control shall be applied, externalSubjectId must be omitted (no acc
+
+The following conventions apply for the endpoint:
+
+- `interface`, `endpointProtocol`, `endpointProtocolVersion`, `subprotocol`, `subprotocolBodyEncoding`, and `securityAttributes` are set as defined in the CX-0002 standard.
```
Is the `CX-002` Standard already publicly available? If so, I think a link would help the reader. 
Potentially all references to the CX standards should link to the source.

mhellmeier replied:
I just added links to the location of the standards.

SebastianBezold commented on docs-kits/kits/Traceability Kit/Software Development View/page_software-development-view.md:
```diff
@@ -386,55 +442,284 @@ If no access control shall be applied, externalSubjectId must be omitted (no acc
+  - `dspEndpoint`: Server and port of the EDC control plane used for contract negotiation.
+
+> :raised_hand: **Backend Data Service for Submodels**
+According to CX-0002, the backend data service identified via `href`and the filter criteria in `subprotocolBody` MUST be conformant to the Asset Administration Shell Profile SSP-003 of the Submodel Service Specification and must at least support the logical operation GetSubmodel. In release 3.2, only the logical parameter Content=Value must be supported via path suffix "/submodel/$value". This mi ...
```
I think here the Tractus-X release is meant, right?
As I understood it, the Standards are published at the Catena-X assosiation, but the models are hosted in Eclipse Tractus-X. 
Could be a good thing to clarify and also share with other teams

mhellmeier replied:
We are talking about the Catena-X release. I updated the section to clarify it.

SebastianBezold commented:
From my point of view just a view minor things, that would not block a merge, but could be considered for further improvement:

- Clarify references to "Catena-X release". In my point of view, there is no Catena-X release and "Tractus-X release" is meant
- Add a few more reference links. I.e. Standards in it's version on catena-x.net (changelog + in text)

SebastianBezold reviewed (approved):
LGTM overall, but cannot provide feedback on the contents, so further approvals should be given

mhellmeier commented:
Based on @SebastianBezold comments:
> * Clarify references to "Catena-X release". In my point of view, there is no Catena-X release and "Tractus-X release" is meant

I updated the files accordingly to only mention "release" instead of "Catena-X release" to reduce misunderstandings.


> * Add a few more reference links. I.e. Standards in it's version on catena-x.net (changelog + in text)

I updated and added some links to the standards. Nevertheless due to the current timeline, it isn't possible to link to the correct standard. As I understand, @danielmiehle will discuss this in the lifecycle process.

mhellmeier commented:
This pull request is still in Draft mode since it waits for internal approval of the traceability use case team as well as a successful quality gate check.

If both things are fulfilled, I will change it to be ready for the merge.

maximilianong reviewed (commented):
Can you please add the versions of the standards you are referencing to
(e.g. CX - 0004 Semantic Model V 1.0.1)

maximilianong reviewed (approved):
After IP clearance - PR will be merged.

mhellmeier commented:
> After IP clearance - PR will be merged.

Just to let you know, IP clearance has been successful. The branch is updated to the current status of `main` accordingly. All kit changes are integrated into the new versioned structure. Ready to be merged, @maximilianong!
